react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+class Pokemons extends React.Component {
+  render() {
+    const listItems = pokedex.map(pokemon =>
+      <li key={pokemon.name}>{pokemon.name}</li>);
```

Comment:
Here it would be better to use the `number` property as the `key`:
```JSX
<li key={pokemon.number}>{pokemon.name}</li>);
```

Reply:
<img width="659" alt="Screen Shot 2022-09-06 at 11 05 49 AM" src="https://user-images.githubusercontent.com/97363006/188707655-83a56e9a-c06a-4987-b088-a6fb4394fd91.png">